### DISCUSSION TITLE
Very slow performance/hanging

### DISCUSSION THREAD

**Discussion by ari14850** [iterate-ch/cyberduck#14582] [Q&A]
Cyberduck 8.5.9  on MacOS 12.6.3 

log in to FTP server via numeric IP, shows directory but then when I double click sub folders it works a few times and then gets stuck "listing directory XXXXXXX"

I do not have this issue using FileZilla so I think the FTP server is ok I have also tried at home and work so I don't think it's an internet connection issue, and I've also replicated with another Mac.

Any tips about logs I can look at to try to see what is happening? I uninstalled Cyberduck but when I reinstalled it still had all my bookmarks etc so I have a feeling I'm not clearing out everything that can be cleared out.

Any advice appreciated.

**Comment by ari14850** [comment]
so is there any other support besides this forum? this seems like a very basic question.

**Comment by tatze96** [comment]
Regarding the uninstall:
Uninstalling Cyberduck won't remove the application support folder, as it is shared between Cyberduck and [Mountain Duck](https://mountainduck.io/). Please refer to our [wiki page](https://docs.cyberduck.io/cyberduck/support/#application-support-folder) for more information concerning where the application support folder is located and what is inside it.

Regarding the listing issue:
Are there multiple thousand items located within the affected folders?

Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the issue, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-14582 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
